litestar/contrib/sqlalchemy/plugins/init/plugin.py

```diff
@@ -2,19 +2,22 @@
 
 from typing import TYPE_CHECKING
 
+from litestar.contrib.sqlalchemy.cli import database_group
```

Comment:
You'll need to put this import inside the `on_cli_init` method, else users will get an `ImportError` when they don't have the CLI dependencies installed